AttendanceSystem.Api/Controllers/SessionsController.cs

```diff
@@ -142,10 +142,51 @@ public async Task<IActionResult> SetAttendance([HttpTrigger(AuthorizationLevel.A
         {
             await _attendanceService.SetStudentAttendance(sessionId, GetUserId(ctx), recordProcessRequest.Kind);
         }
-        
         return new NoContentResult();
     }
-    
+
+    [Function($"{nameof(SessionsController)}-{nameof(SetAttendance)}")]
+    public async Task<ActionResult<Session>> SetAttendance([HttpTrigger(AuthorizationLevel.Anonymous, "put", Route = "courses/{courseId}/sessions/{sessionId:guid}/attendance")] HttpRequest req, [SwaggerIgnore] FunctionContext ctx, string courseId, Guid sessionId, [FromBody] List<UpdateAttendanceContract> contracts)
```

Comment:
Did you try this out? I would expect swagger to reject generating the OpenAPI spec because you have a duplicate method.

Why is there a duplicate method? I already updated the `UpdateAttendanceContract` to have a list of contracts. I think you can merge this method with the one above, or at least only use one `UpdateAttendanceContract`.